errata_tool/erratum.py

```diff
@@ -722,7 +754,7 @@ def setFileInfo(self, file_info):
         self._processResponse(r)
 
     def removeBuilds(self, buildlist):
-        if type(buildlist) is not str and type(buildlist) is not list:
+        if not isinstance(buildlist, str) and not isinstance(buildlist, list):
```

Comment:
different issue but str here should probably be six.string_types.  Will open separate issue for this

Reply:
https://github.com/red-hat-storage/errata-tool/issues/181

Reply:
sorry for noise not a problem, verified it was correct behavior.